documentation/content-standards.md

```diff
@@ -126,13 +126,21 @@ Currently, Codebytes supports the following languages:
 | `CatalogContent` | Slugs of Codecademy course and Path landing pages that relate to the entry. We're storing all of our slugs in the [catalog-content.md file](https://github.com/Codecademy/docs/blob/main/documentation/catalog-content.md).<br /> - Please avoid linking to individual content items, because their URLs may change and some are gated by Pro membership.<br /> - We aim to include 2 slugs with every entry (1 free course and 1 Pro course/Path). | CatalogContent:<br /> - "introduction-to-javascript"<br /> - "paths/front-end-engineer-career-path" |
 
 #### General Writing Tips
-
+- Be clear and concise
```

Comment:
```suggestion
```

I think this could be incorporated with the exiting tip beginning with "Brevity without sacrificing..."